src/tutorials/app.md

```diff
@@ -73,7 +73,7 @@ This command will run
 - a Cozy `stack` using docker (the image `cozy/cozy-app-dev`) to serve your application inside it.
 - The same docker image starts a CouchDB containing your data (be careful, since a data volume is not attached by default, your data will disappear on restart. You should start the docker image manually if you want to persist your data).
 
-Your application will be available at http://<MY_APP_SLUG>.cozy.tools:8080. Password is `cozy`.
+Your application will be available at http://<MY_APP_SLUG>.cozy.tools:8080. Password is `cozy`. Since Hot Module Replacement is used by default, you'll need to disable the CORS check for your browser. For instance, on Chrome OSX : `open -n -a /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --args --user-data-dir="/tmp/chrome_dev_test" --disable-web-security` (/!\ Be careful, by doing it you remove an essential security check. Use it just on your localhost).
```

Comment:
Nit: It would be a bit easier if the line was in a code block, instead of an inline code block.

I am not sure that we should be advocating this method. It would be simpler if cozy/cozy-app-dev started with CORS disabled. I personally use `--disable-csp` when starting the stack to do this.

Reply:
csp !== cors 

https://stackoverflow.com/a/50726191

Reply:
The error I have in my browser when I do not start the stack with `--disable-csp` is 

```
Content Security Policy: Les paramètres de la page ont empêché le chargement d’une ressource à http://localhost:8282/main/banks.css (« style-src »).
```

When I add this flag, the loading works and hot reloading works. Are you sure this is CORS that block the loading of the request ?

Reply:
I am not sure that we should recommend the docker image. Since we do not use it, it's hard
to understand precisely what's going on when a contributor has a problem.

I suggest to tell our contributors to run the stack from the source, with a running CouchDB.

I am putting below a starting point for this, I'd be glad to know what are your
thoughts on this.

---

First, download a binary of the `stack`, the back-end server that interacts
with the CouchDB database and serves the applications.

We recommend to run the stack from the master branch and compile it.

```
# Install a working go environment, we recommend go@1.14.4
# Brew is mac os x only, on Linux, use your distribution's package manager
brew install golang
git clone git://github.com/cozy/cozy-stack.git
cd cozy-stack
go install
```

At this point you have access to the `cozy-stack` binary.

```
$ cozy-stack serve --help
Starts the stack and listens for HTTP calls
It will accept HTTP requests on localhost:8080 by default.
Use the --port and --host flags to change the listening option.
...
```

You still need to have a running CouchDB. Below are the official docs from CouchDB
to install and run a CouchDB instance. We do not support yet CouchDB@3, so be
careful to install CouchDB@2.3.1

- [Instructions to install CouchDB on macOS](https://docs.couchdb.org/en/2.3.1/install/mac.html)
- [Instructions to install CouchDB on unix like systems]https://docs.couchdb.org/en/2.3.1/install/unix.html)

Once couchDB is installed and running, you can launch the `cozy-stack`.
In development, the cozy-app-dev.sh script from the `cozy-stack` repository must
be used to launch the stack. It will automatically create a cozy if you do not have
one, and create the necessaray databases.

Now, you need to tell the `stack` where it should load your app from. To do this,
we'll use a folder containing symlinks to the build folders of our apps. The 
`cozy-stack` will be started with the `-d` flag that configures this setting.

Let's create this directory and one symlink to an app. In the example below,
you have created an app through CCA into the `test-app` directory.

```
$ create-cozy-app test-app
# Fill the information asked and wait until completion
$ cd test-app
# Build the app into the `build` directory
$ yarn build
```

Now, we can start the cozy-stack.

```bash
cd ~/code/cozy # the directory where you keep all your cozy related folder
mkdir cozy-apps
ln -s ~/code/cozy/test-app/build cozy-apps/testapp
cd ~/code/cozy-stack

# Starts the stack, test-app must have been built once for the stack to
# register correctly the app
./scripts/cozy-app-dev.sh ../cozy-apps
```

We suggest to have a bash alias that starts the stack in this way.

At this point, the stack serves your app, you can go to http://testapp.cozy.tools:8080
to see the results.

To modify the code of the app, you can start a development server from the app.

```
yarn start
```